packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -379,3 +431,24 @@ export interface AddCapacityOptions extends AddAutoScalingGroupCapacityOptions,
    */
   readonly instanceType: ec2.InstanceType;
 }
+
+export interface NamespaceOptions {
+  /**
+   * The domain name for the namespace, such as foo.com
+   */
+  readonly name: string;
+
+  /**
+   * The type of CloudMap Namespace to create in your cluster
+   *
+   * @default PrivateDns
+   */
+  readonly type?: cloudmap.NamespaceType.DnsPrivate | cloudmap.NamespaceType.DnsPublic;
```

Comment:
I wonder if it makes sense to turn this into a boolean? Saves an import if you want to configure it.

Reply:
using enum!